zenhappens/src/book-appointment.html

```diff
@@ -0,0 +1,161 @@
+        <label class="form__radio--label" for="organic-facials"><span>Organic Facials</span></label>
+
+        <input class="form__radio--input u-sr-only" id="waxing" type="radio" name="service" value="waxing">
+        <label class="form__radio--label" for="waxing"><span>Waxing</span></label>
+
+        <input class="form__radio--input u-sr-only" id="reiki" type="radio" name="service" value="reiki">
+        <label class="form__radio--label" for="reiki"><span>Reiki</span></label>
+
+        <input class="form__radio--input u-sr-only" id="cupping" type="radio" name="service" value="cupping">
+        <label class="form__radio--label" for="cupping"><span>Cupping</span></label>
+
+        <input class="form__radio--input u-sr-only" id="home-massage" type="radio" name="service" class="u-sr-only" value="home massage">
+        <label class="form__radio--label" for="home-massage"><span>Home Massage</span></label>
+        </div>
+
+        </fieldset>
+
+        <fieldset class="js-massage__questions" hidden>
+        <legend class="appointment__form--question">What type of Massage?</legend>
+        
+        <div class="form__radio-container">
+        
+        <input class="form__radio--input u-sr-only" id="basic" type="radio" name="massage type" value="basic" checked>
+        <label class="form__radio--label" for="basic">Basic</label>
+
+        
+        <input class="form__radio--input u-sr-only" id="custom" type="radio" name="massage type" value="custom">
+        <label class="form__radio--label" for="custom">
+        <span>Custom</span>
+        </label>
+
+        <input class="form__radio--input u-sr-only" id="thai-massage" type="radio" name="massage type" value="thai massage">
+        <label class="form__radio--label" for="thai-massage">
+        <span>Thai Massage</span>
+        </label>
+
+        <input class="form__radio--input u-sr-only" id="prenatal-massage" type="radio" name="massage type" value="prenatal massage">
+        <label class="form__radio--label" for="prenatal-massage">
+        <span>Prenatal Massage</span>
+        </label>
+
+        <input class="form__radio--input u-sr-only" id="foot-massage" type="radio" name="massage type" value="foot massage"> 
+        <label class="form__radio--label" for="foot-massage"><span>Foot Massage</span></label>
+
+        <input class="form__radio--input u-sr-only" id="couples-massage" type="radio" name="massage type" value="couples massage">
+        <label class="form__radio--label" for="couples-massage"><span>Couples Massage</span></label>
+        </div>
+        </fieldset>
+
+        <fieldset class="js-massage__questions" hidden>
+        <legend class="appointment__form--question">For how long?</legend>
+        
+        <div class="u-flex">
+        {# best way to center range input #}
+        <div class="form__range-container">
+        
+        <div class="range">
+        <input type="range" min="1" max="7" steps="1" value="1">
```

Comment:
These aria labels work along side the JS changes to make this input more accessible. 🙌 
```suggestion
        <input type="range" min="1" max="7" steps="1" value="1" aria-valuemin="1" aria-valuemax="7" aria-valuenow="1" aria-valuetext="15 minutes">
```